### ISSUE TITLE
Can't subscribe to characteristic

### ISSUE THREAD

**Issue by dhinojosac** [rohitsangwan01/win_ble#31]
Hi, 

When attempting to subscribe to a characteristic and not receiving updates, an error is thrown stating "The attribute cannot be written".

![image](https://github.com/rohitsangwan01/win_ble/assets/5265771/f5628ddb-0d48-4547-86fd-c2c7b373d18a)

![image](https://github.com/rohitsangwan01/win_ble/assets/5265771/7587826b-5a72-4a07-a2a7-2e3aabf04328)

**Comment by rohitsangwan01** [comment]
@dhinojosac can you cross check in other apps like `NrfConnect` with this same characteristic

**Comment by rohitsangwan01** [comment]
@dhinojosac usually we have to write to the descriptor to turn on notification/indication, [here](https://github.com/rohitsangwan01/win_ble/blob/main/BLEServer/BLEServer/BLEServer.cpp#L407C42-L407C95) is the logic

**Comment by rohitsangwan01** [comment]
@dhinojosac yes like mentioned in the post you shared [here](https://stackoverflow.com/a/69126661) you need to add a descriptor to the characteristic which needs to be notified, That descriptor is responsible for starting or stoping notification events on the Ble server side, 
So basically we write start/stop commands to that descriptor and based on the success result we subsribe to the notification updates of that characteristics
